css/styles.css

```diff
@@ -0,0 +1,417 @@
+* {
+  box-sizing: border-box;
+  font-family: Poppins, sans-serif;
+}
+
+header {
+  background-color: #f84453;
+  display: flex;
+}
+
+body {
+  background-color: #0f141b;
+}
+
+h2 {
+  margin-left: 1rem;
+  color: white;
+}
+
+h2 > a {
+  color: white;
+  text-decoration: none;
+}
+
+header > img {
+  padding-left: 1rem;
+}
+
+.fa-bars {
+  color: white;
+}
+
+.search-box {
+  border: none;
+  border-radius: 0.6em;
+  background-color: #5c5f74;
+  width: 80%;
+  height: 250px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 3.5rem;
+  margin-left: 3rem;
+  margin-bottom: 3rem;
+}
+
+.red-box > img {
+  width: 55px;
+  height: 60px;
+  padding: 0.3rem;
+}
+
+.red-box {
+  background-color: #f84453;
+  width: 70px;
+  height: 70px;
+  border-radius: 0.5em;
+  border: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: 105px;
+  margin-bottom: 3rem;
+}
+
+.search-box-content > input,
+.search-box-content > button {
+  border-radius: 0.3em;
+  border: none;
+}
+
+.search-box-content > input {
+  padding: 6px 12px;
+  width: 200px;
+  height: 36px;
+}
+
+.search-box-content > button {
+  background-color: #f84453;
+  color: white;
+  margin-left: 0.5rem;
+  padding: 0;
+  width: 75px;
+  height: 36px;
+}
+
+.text {
+  color: white;
+  background-color: #f84453;
+  text-align: center;
+  padding-top: 0.2rem;
+  padding-bottom: 0.2rem;
+}
+
+.carousel {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #868686;
+}
+
+.carousel-container {
+  margin-top: 3rem;
+}
+
+.carousel-item > img {
+  width: 100%;
+  border-radius: 0.3em;
+  position: relative;
+  left: 8px;
+}
+
+.carousel-buttons {
+  background-color: #868686;
+  display: flex;
+  justify-content: center;
+}
+
+.circle-button {
+  border-radius: 50%;
+  border: none;
+  height: 1rem;
+  width: 1rem;
+  margin: 0.5rem 0.6rem 1.5rem;
+  cursor: pointer;
+}
+
+.button-right,
+.button-left {
+  background-color: #868686;
+  border: none;
+  height: 100%;
+  color: white;
+  margin-top: 1.5rem;
+  font-size: 2rem;
+  cursor: pointer;
+}
```

Comment:
You should add a `transition` so that when you change the `color` on hover, it won't jump